src/app/components/iterations-panel/iterations-panel.component.html

```diff
@@ -1,217 +1,54 @@
+    <!-- display the active iterations first -->
+
+    <!-- Nested Iteration Display -->
     <div class="iterations">
-      <div class="iteration-header pointer past-iteration-header" [dragula]="'wi-bag'"
-        (click)="isCollapsedPastIteration = !isCollapsedPastIteration">
-        <h3>
-          <i class="fa fa-icon pointer"
-              [ngClass]="{'fa-angle-right': isCollapsedPastIteration,
-                          'fa-angle-down': !isCollapsedPastIteration}"></i>
-          Past Iterations
-        </h3>
-      </div>
-      <ul class="not-current-iteration"
-          [collapse]="isCollapsedPastIteration">
-        <li class="iteration-subheader iteration-container" *ngFor="let iteration of closedIterations" (click)='getWorkItemsByIteration(iteration)' [routerLinkActive]="'selected-iteration'"
-            [routerLink]="[]"
-            [queryParams]="{q: constructURL(iteration.id)}"
-            [dragula]="'wi-bag'" [attr.data-id]="iteration.id">
-          <div class="pull-right">
-            <span class="badge">{{iteration.relationships?.workitems?.meta?.total}}</span>
-            <div *ngIf="loggedIn" class="dropdown-kebab-pf dib dropdown" dropdown>
-              <button class="btn btn-link dropdown-toggle" (click)="kebabMenuClick($event)"
-                      type="button"
-                      aria-haspopup="true"
-                      aria-expanded="true"
-                      dropdownToggle>
-              <i class="fa fa-ellipsis-v"></i>
-              </button>
-              <ul class="dropdown-menu-right dropdown-menu" *dropdownMenu>
-                <li>
-                  <a class="pointer" (click)="modal.openCreateUpdateModal('update', iteration, $event);">
-                    Edit
-                  </a>
-                </li>
-                <li>
-                  <a class="pointer" (click)="modal.openCreateUpdateModal('start', iteration, $event);">
-                    Start
-                  </a>
-                </li>
-                <li>
-                  <a class="pointer" (click)="modal.openCreateUpdateModal('createChild', iteration, $event);">
-                    Create child
-                  </a>
-                </li>
-              </ul>
-            </div>
-          </div>
-          <div class="truncate">
-            <strong class="iteration-name truncate"
-              tooltip="{{iteration.attributes.name}}"
-              placement="top" id="{{iteration.attributes.name}}">
-              {{iteration.attributes.name | truncate : 40 }}
-            </strong>
-            <div class="parent-iteration">
-              <span tooltip="{{ iteration.attributes.resolved_parent_path + '/' + iteration.attributes.name }}"
-                placement="top">
-                {{ iteration.attributes.resolved_parent_path + '/' + iteration.attributes.name | truncate : -35 }}
-              </span>
-            </div>
-          </div>
-        </li>
-      </ul>
+      <alm-tree-list #treeList
```

Comment:
FYI, we just created a new patternfly-ng tree list, based on the latest pf-list layout from Patternfly. It has the following advantages.

- Instead of Angular 2, the underlying angular-tree-component now uses Angular 4
- It's based on angular-tree-component v4.10 instead of 2.7.0
- It used the latest pf-list styles from PatternFly, so your issues with 'list-group-item' should go away
- Supports additional features like checkboxes, multiple row selection, etc.

You will also want to use the pfng-tree-list component because the tree-list will be removed from ngx-widgets.

See: https://github.com/patternfly/patternfly-ng/pull/120

Reply:
Now that I think of it some more, I don't believe we should be using a tree list here at all. Using the tree list component is overkill here. There is no drag and drop, no tree hierarchy, etc. We don't need to override all those tree lists styles and methods.

Basically, you only need a standard HTML list and some indentation. 

@mindreeper2420 suggested using the Bootstrap collapse, which I believe is a much better idea than using the tree list component.